oc-aliases.sh

```diff
@@ -80,29 +87,29 @@ function load_connection() {
 		echo "load_connection <connection.json>"
 	else
 		gettoken
-		curl -k -H "Authorization: Bearer ${TOKEN}" https://${SD_HOST:?environment variable must be set}/connmgr/v1/connections -X POST -H "Content-type: application/json" -d@"$1"
+		curl -k -H "Authorization: Bearer ${TOKEN}" https://${SD_CONSOLE_ROUTE:?environment variable must be set}/connmgr/v1/connections -X POST -H "Content-type: application/json" -d@"$1"
 	fi
 }
 
 # Get all running pods
-alias allpods='oc get pods --all-namespaces'
-alias kp='oc get pods --all-namespaces'
+alias allpods='oc -n ${PROJECT_NAME} get pods'
+alias kp='oc -n ${PROJECT_NAME} get pods'
 
 # Fetch a valid authentication token
-alias gettoken="export TOKEN=\$(curl -k -u \${SD_USER:?environment variable must be set}:\${SD_PASSWORD:?environment variable must be set} https://\${SD_HOST:?environment variable must be set}/auth/v1/token -I | grep X-Auth-Token |cut -f 2 -d \" \")"
+alias gettoken='export TOKEN=$(curl -s -k -u ${SD_USER:-sdadmin}:${SD_PASSWORD:?environment variable must be set} https://${SD_CONSOLE_ROUTE:?environment variable must be set}/auth/v1/token -I | grep -i X-Auth-Token | cut -f 2 -d " ") | xargs'
```

Comment:
Once defined SD_CONSOLE_ROUTE, you can remove the ":?environment variable must be set"